site/en/reference/cli/concepts/udp.md

```diff
@@ -0,0 +1,193 @@
+# Send UDP Messages with OT CLI
+
+Thread networks offer several `udp` commands for establishing peer-to-peer communication
+using UDP sockets. Each peer needs its own socket.
+
+In addition to opening the required sockets, one requirement is that the socket
+on the receiving end of the initial communication be bound.
+
+The examples that follow demonstrate how you can open and bind sockets, how to
+to specify a peer with which to associate a local socket, and how to send messages
+using UDP sockets.
+
+## UDP Commands
+
+For a list of `udp` commands, type `help`:
+
+> udp help
+bind
+close
+connect
+linksecurity
+open
+send
+Done
+
+### `open` command
+
+Use the `udp open` command to open a socket to begin UDP communication.
+You then have the option to bind the socket to a specific IP address and port.
+
+### `bind` command
+
+After you `open` a socket, you can run a `udp bind` command to assign an IPv6 address
+and a port to the open socket. This binds the socket for communication. Assigning the
+IPv6 address and port is also referred to as naming the socket. Binding is required to
+initially receive a UDP message. However, binding a socket is not required
+on the node that is sending the first message. If the initiating node opens but
+does not bind a socket, the socket will choose its own local IP address and ephemeral port
+when sending the message.
+
+### `connect` command
+
+A `udp connect` command can be used to establish UDP communication with one specific peer socket
+by specifying the IP address and UDP port number of the peer. This command can later be
```

Comment:
Suggest to simplify:
`udp connect` can be used to connect the example socket to a peer socket address.